source/shared/net/guides/become-baker.rst

```diff
@@ -80,11 +84,17 @@ and send a ``baker add`` transaction to the network:
 where you replace
 
 - ``<amount-to-stake>`` with the CCD amount for the baker's initial stake
-- ``<concordium-data-dir>`` with any path of your choice. Remember to configure your node to bake using this path:
+- ``<concordium-data-dir>`` with:
+
+  * on Linux and MacOS: ``~/.local/share/concordium``
+  * on Windows: ``%LOCALAPPDATA%\\concordium``.
```

Comment:
I might be confused, but these paths are only relevant for the **docker** distribution.

For the other ones they can be arbitrary, as it said before.